### ISSUE TITLE
Using cusom linters/formatters for ts/scss/html

### ISSUE THREAD

**Issue by sdurnov** [nrwl/nx#414]
Sadly, but prettier plugins for VSCode are extremely buggy. Prettier ignores turning it off for certain languages, breaks other formatters for all languages.
Please provide option to use custom formatters and linters for html/scss/ts

**Comment by Springrbua** [comment]
I agree with this. I had to turn of the formatting of prettier, as it does not conform to our ruels and causes TSLint errors.
As prettier will probably never be very configurable (_Prettier is an opinionated code formatter_) it would be cool, if you can decide which formatter to use.
At the moment, I have to use multiple tools, in the right order, to get the code-style I want.
